README.md

```diff
@@ -122,7 +122,11 @@ query Sources {
 ```graphql
 # Query Cultuurhistorische Thesaurus (CHT)
 query {
-  terms(sources: ["https://data.cultureelerfgoed.nl/PoolParty/sparql/term/id/cht"], query: "fiets") {
+  terms(
+    sources: ["https://data.cultureelerfgoed.nl/PoolParty/sparql/term/id/cht"],
+    query: "fiets",
+    queryMode: SMART   # Forwards-compatible query type.
```

Comment:
1. Is `queryMode` a good name for this parameter?
2. Is `SMART` a good name for its value, i.e. the forwards-compatible mode where pre-processing is done in TypeScript rather than SPARQL.

Reply:
1. `queryMode` sounds good! Alternatively, `queryType`?
2. I'm okay with `SMART`. Alternatively, `AUTO` ('automatically'), a little less 'pompous'?

Reply:
1. I previously had `queryType` but considered the word `type` confusing in the context of a GraphQL schema because in GraphQL (query) types mean something else.
2. Yeah, `SMART` sounds smarter than it really is. 😁 `AUTO`, on the other hand, is somewhat vague. Perhaps `OPTIMIZED`?

Reply:
Renamed to `OPTIMIZED`.